lib/mixduty/response.ex

```diff
@@ -0,0 +1,69 @@
+defmodule Mixduty.Response do
+  require Logger
+  alias Mixduty.Error
+
```

Comment:
nit:
```suggestion
  alias Jason, as: JSON

```
And use `JSON.decode()` instead of `Jason.decode()`